keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/orca/container/ContainerJobConfig.kt

```diff
@@ -0,0 +1,151 @@
+package com.netflix.spinnaker.keel.orca.container
+
+import com.netflix.spinnaker.keel.orca.container.ImageVersionReference.*
+
+const val RUN_JOB_TYPE : String = "runJob"
+
+/**
+ * Containers can be identified by either:
+ *  tag (e.g., "latest")
+ *  digest (e.g., "sha256:780f11bfc03495da29f9e2d25bf55123330715fb494ac27f45c96f808fd2d4c5")
+ */
+enum class ImageVersionReference {
+  TAG,
+  DIGEST
+}
+
```

Comment:
nit: It's weird that `ContainerJobConfig(` is a data class but the parameters have newlines in between them.

Reply:
> nit: It's weird that `ContainerJobConfig(` is a data class but the parameters have newlines in between them.

Fun fact: `ContainerJobConfig`: started out its life as an interface, and then morphed into a data class before I submitted the PR, but I forgot to clean up the spacing.

Reply:
Fixed.